### PR TITLE
dart:ui - drop deprecated hash functions

### PR DESCRIPTION
**Pull request by kevmoo**
These were deprecated a LONG time ago

### CODE REVIEW

File: lib/ui/experiments/ui.dart [hunk 1]
```diff
@@ -45,8 +44,7 @@ part '../platform_isolate.dart';
 part '../plugins.dart';
 part '../pointer.dart';
 part '../semantics.dart';
-part 'setup_hooks.dart';
 part '../text.dart';
 part '../window.dart';
-
 part 'scene.dart';
+part 'setup_hooks.dart';
```
Comment (kevmoo) [inline lib/ui/experiments/ui.dart:50]: just a tiny (automatic, but unrelated) sort of the parts

**Comment by stuartmorgan** [comment]
test-exempt: removing code

**Review by jonahwilliams — APPROVED** [review]
LGTM

It seems like 3.1.X was long enough ago!

**Comment by jiahaog** [comment]
Reason for revert: There are still internal users of these hash functions (b/352191023). Flutter also exports it [here](https://github.com/flutter/flutter/blob/72f83d3237a051a8b1b849abd267d4a4e80ff774/packages/flutter/lib/src/painting/basic_types.dart#L58) so its easy for developers to use them even without importing `dart:ui`. Though it's an easy fix, I'd imagine this to be a breaking change for the OSS ecosystem as well.

**Pull request merged**